test/acceptance/behave/README.md

```diff
@@ -21,15 +29,11 @@ Note: We recommend the use of virtualenv, because is an isolated working copy of
 - Both if you are using a virtual environment or not:
   * Change to the test/acceptance/behave folder of the project.
   * Install the requirements for the acceptance tests in the virtual environment
+  * You should add "--upgrade" (if you have a previous installed version of this library)
 ```
-     pip install -r requirements.txt --allow-all-external
+     pip install [--upgrade] -r requirements.txt --allow-all-external
 ```
 
-#### Requirements to fabric (http://www.fabfile.org/)
-```
-     yum install gcc python-devel
-```
-   Fabric is a Python (2.5-2.7) library and command-line tool for streamlining the use of SSH for application deployment or systems administration tasks.
 
 #### Folders/Files Structure
 
```

Comment:
Documentation says:

```
logs/: folder to log (it is created if does not exist)
```

However, I get this error if the directory doesn't exist:

```
(venv)fermin@oriondeb:~/src/fiware-orion/test/acceptance/behave$ behave components/ngsiv2/entities/general_operations.feature 
...
IOError: [Errno 2] No such file or directory: '/home/fermin/src/fiware-orion/test/acceptance/behave/logs/behave.log'
```

Once the logs/ directory is created, the problem disapears.